data/read-only.sql

```diff
@@ -0,0 +1,2 @@
+-- Sample SQL file to make tables read only
+REVOKE INSERT, UPDATE, DELETE, TRUNCATE ON ALL TABLES IN SCHEMA public FROM anon;
```

Comment:
@mogul I don't think this SQL script will work as intended without the anonymous role being set via the `vars.yml` file. So the anonymous role in that file would be the one referenced here.